client/app/components/QueryEditor.jsx

```diff
@@ -102,9 +102,11 @@ class QueryEditor extends React.Component {
     this.onLoad = (editor) => {
       // Release Cmd/Ctrl+L to the browser
       editor.commands.bindKey('Cmd+L', null);
-      editor.commands.bindKey('Ctrl+P', null);
```

Comment:
I forgot about the Parameter dialog before :sweat_smile:. I tested here again, if this line is missing, it stops working...
Either keep this line or using `editor.commands.bindKey({ win: 'Ctrl+P', mac: null }, null);` make the dialog shortcut functional again.